lib/src/create_report.dart

```diff
@@ -335,120 +302,142 @@ Future<ReportSection> _trustworthDependency(
-            ),
-          );
-          allowsCurrentFlutter = false;
+      final usesFlutter = pubspec.usesFlutter;
+      final flutterDartVersion = Version.parse(
+          (flutterVersions['dartSdkVersion'] as String).split(' ').first);
+      final allowsCurrentFlutterDart =
+          sdkConstraint?.allows(flutterDartVersion) ?? false;
+
+      if (!allowsCurrentFlutterDart) {
+        issues.add(
+          _Issue(
+            'The current sdk constraint does not allow the dart version latest of Flutter ($flutterDartVersion)',
+            span: _tryGetSpanFromYamlMap(pubspec.environment, 'sdk'),
+          ),
+        );
+      } else {
+        if (usesFlutter) {
+          // TODO(sigurdm): this will not work well locally (installed version will
+          // not be latest). Perhaps we should query somewhere for the latest version.
+          final currentFlutterVersion =
+              Version.parse(flutterVersions['frameworkVersion'] as String);
+          final flutterConstraint = pubspec.flutterSdkConstraint;
+          if (!flutterConstraint.allows(currentFlutterVersion)) {
+            issues.add(
+              _Issue(
+                'The current flutter constraint does not allow the latest Flutter ($currentFlutterVersion)',
+                span: _tryGetSpanFromYamlMap(pubspec.environment, 'flutter'),
+              ),
+            );
+          }
         }
       }
     }
+    final status = issues.isEmpty ? _Status.good : _Status.bad;
+    final points = issues.isEmpty ? 10 : 0;
+    return _Subsection(
+      'Package supports latest minor version of the Dart and Flutter sdks',
+      issues,
+      points,
+      status,
+    );
   }
 
-  String publisher;
+  Future<_Subsection> publisher() async {
+    final issues = <_Issue>[];
+    String publisher;
+
+    try {
+      publisher = json.decode(await read(
+              'https://pub.dev/api/packages/${Uri.encodeComponent(pubspec.name)}/publisher'))[
+          'publisherId'] as String;
+    } on ClientException catch (e) {
+      issues.add(_Issue(
+        'Could not retrieve publisher information. Has this package been published before? ($e)',
+      ));
+    }
 
-  try {
-    publisher = json.decode(await read(
-            'https://pub.dev/api/packages/${Uri.encodeComponent(pubspec.name)}/publisher'))[
-        'publisherId'] as String;
-  } on ClientException catch (e) {
-    issues.add(_Issue(
-      'Could not retrieve publisher information. Has this package been published before? ($e)',
-    ));
+    if (publisher == null) {
+      issues.add(_Issue('Package is not published under a verified publisher.',
+          suggestion: 'See $_publisherDoc for more information.'));
+    }
+    final status = issues.isEmpty ? _Status.good : _Status.bad;
+    final points = issues.isEmpty ? 10 : 0;
+    return _Subsection(
+      'Package supports latest minor version of the Dart and Flutter sdks',
```

Comment:
This is the wrong summary text. I think something like 'Publish using a verified publisher'.